.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: build-all-configs
+on: [push]
+jobs:
+  build-on-osx-for-osx:
+    runs-on: macos-10.15
+    steps:
+    - uses: actions/checkout@v2
+    - id: configure-cmake
+      run: cmake -S . -B build -DDJINNI_WITH_OBJC=ON -DDJINNI_STATIC_LIB=ON -G Xcode
+    - id: build-release
+      run: cmake --build build --parallel $(sysctl -n hw.ncpu) --config Release
+    - id: install-files
+      run: cmake --build build --parallel $(sysctl -n hw.ncpu) --config Release --target install -- DESTDIR=zinstall_root
+    - id: list-installed-files
+      run: cd build/zinstall_root && du -a | tail -r | awk -F ' ' '{print $2}'
```

Comment:
What's that output for? Is it meant for debugging the step `test-file-list`?

Reply:
this is, if the test fails, you want to know what is in the folder. 
if this output would not be, you would just have a failing test, but not the data (folder content) that caused the failing test. And this makes reasoning hard...